civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -722,3 +723,7 @@ $carousel-box-shadow-color: rgba(#3f3c55,0.3);
     }
   }
 }
+
+.select2-container .select2-choice .select2-arrow {
+  width: 26px !important;
```

Comment:
This is a too generic rule for this file, it should go for example to `civihr_default_theme/scss/components/_datepicker.scss`

Reply:
This was left while debugging! already removed!